chrome/dist/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Local Home SDK App</title>
+  <script type="module" src="main.js"></script>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+  <div id="file-picker-wrapper">
+    <label for="app-file-picker"><b>Load App Bundle: </b></label>
+    <input id="app-file-picker" type="file">
+  </div>
+  <form action="javascript:localHomePlatform.onUdpScanButton();">
+    <div class="container">
+      <hr>
+      <h2>UDP Scan</h2>
+      <hr>
+
+      <label for="udp-request-id"><b>Request ID</b></label>
+      <input type="text" id="udp-request-id" required>
+
+      <label for="udp-device-id"><b>Device ID</b></label>
+      <input type="text" id="udp-device-id" required>
+
+      <label for="udp-broadcast-address"><b>Broadcast Address</b></label>
+      <input type="text" id="udp-broadcast-address" required>
+
+      <label for="udp-broadcast-port"><b>Broadcast Port</b></label>
+      <input type="text" id="udp-broadcast-port" required>
+
+      <label for="udp-listen-port"><b>Listen Port</b></label>
+      <input type="text" id="udp-listen-port" required>
+
+      <label for="udp-discovery-packet"><b>Discovery Packet</b></label>
+      <input type="text" id="udp-discovery-packet" required>
+
+      <button type="Start Scan" class="submit-button">Submit</button>
```

Comment:
For a11y, consider naming the submit buttons more explicitly. "Start Scan" vs "Sent Intent" - makes it easier to navigate the page with a screen reader.

Reply:
Good point. Done.